rubicon_ml/cli.py

```diff
@@ -78,7 +80,78 @@ def ui(root_dir, page_size, project_name, host, port, debug, storage_options):
     dashboard.serve(run_server_kwargs=run_server_kwargs)
 
 
-# CLI groups
+@cli.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+    )
+)
+@click.option(
+    "--root-dir",
+    type=click.STRING,
+    default=os.environ.get("RUBICON_ROOT_DIR"),
+    help="The absolute path to the top level folder holding the rubicon-ml project.",
+    required=False,
+)
+@click.option(
+    "--project-name",
+    default=os.environ.get("RUBICON_PROJECT_NAME"),
+    type=click.STRING,
+    help="Name of rubicon project to retrieve experiments from",
+    required=False,
+)
+@click.option(
+    "--color",
+    "-c",
+    type=click.STRING,
+    help="Color of query output",
+    required=False,
+)
+@click.argument(
+    "query",
+    nargs=1,
+    required=True,
+)
+def search(root_dir, project_name, color, query):
+    """
+    This function provides capability to query  rubicon experiments from the command line without using python
+    directly and output the JSON in the command line.
+    """
+
+    proj_name_env = "RUBICON_PROJECT_NAME"
+    root_dir_env = "RUBICON_ROOT_DIR"
+    fg_color = "bright_yellow"
+
+    if (proj_name_env not in os.environ and project_name is None) or (
+        root_dir_env not in os.environ and root_dir is None
```

Comment:
I think the first part of each of the and clauses here are unnecessary, right? like if `proj_name_env` isn't in `os.environ` then `project_name` will be `None` since `os.environ.get("RUBICON_PROJECT_NAME")` returned `None` for the default - so we can just check `if project_name is None` and it'll achieve the same thing